logs/build.go

```diff
@@ -13,6 +15,11 @@ type buildCmd struct {
 }
 
 func (cmd *buildCmd) Run(ctx context.Context, client *api.Client) error {
+	var build apps.Build
+	if err := client.Get(ctx, api.NamespacedName(cmd.Name, client.Project), &build); err != nil {
+		return err
+	}
```

Comment:
This will break querying build logs by app name. So we need to first check if `cmd.Name` is not empty.

Reply:
added a check